ec2setup/algorithms/LoadForecasting/LoadForecastingRunner.py

```diff
@@ -33,9 +33,10 @@
 energy_clip = 100  # max battery size to allow drawing from the energy distributions
 
 # Create and Edit Configuration
-config = OngoingConfig(num_evs, aggregation_level=aggregation_level, county=county_choice, fast_percent=fast_percent, 
-                               work_percent=work_percent, res_percent=res_percent, l1_percent=l1_percent, publicl2_percent=publicl2_percent,
-                               res_daily_use=res_daily_use, work_daily_use=work_daily_use, fast_daily_use=fast_daily_use, rent_percent=rent_percent)
+config = OngoingConfig(num_evs, aggregation_level=aggregation_level, county=county_choice, res_percent=res_percent,
+                                fast_percent=fast_percent, publicl2_percent=publicl2_percent, work_percent=work_percent,
+                                l1_percent=l1_percent, rent_percent=rent_percent,
+                                res_daily_use=res_daily_use, work_daily_use=work_daily_use, fast_daily_use=fast_daily_use)
```

Comment:
changed the order of inputs as it's easier to follow what's happening in loadForecastRunner